resources/public/api/conf/1.0/errors.raml

```diff
@@ -158,4 +158,115 @@ traits:
+                   responses:
+                     400:
+                       body:
+                         application/json:
+                           type: types.errorResponse
+                           examples:
+                             formatProviderAddress:
+                               description: The format of providers address is invalid.
+                               value:
+                                 code: FORMAT_PROVIDERS_ADDRESS
+  ruleBenefit:
+                   responses:
+                     400:
+                       body:
+                         application/json:
+                           type: types.errorResponse
+                           examples:
+                             ruleBenefitInExcess:
+                               description: Provision of either Lump sum benefit taken in excess of lifetime allowance or Benefit in excess of lifetime allowance.
+                               value:
+                                 code: RULE_BENEFIT
+  ruleIncorrectOrEmptyBodySubmitted:
+                   responses:
+                     400:
+                       body:
+                         application/json:
+                           type: types.errorResponse
+                           examples:
+                             ruleIncorrectOrEmptyBodySubmitted:
+                               description: An empty or non-matching body was submitted.
+                               value:
+                                 code: RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED
+  matchingResourceNotFound:
+                   responses:
+                     404:
+                       body:
+                         application/json:
+                           type: types.errorResponse
+                           examples:
+                             matchingResourceNotFound:
+                               description: The supplied income source could not be found.
+                               value:
+                                 code: MATCHING_RESOURCE_NOT_FOUND
+
+  formatQopsRef:
+                   responses:
+                     400:
+                       body:
+                         application/json:
+                           type: types.errorResponse
+                           examples:
+                             formatQopsRef:
+                               description: The format of QOPS reference number is not valid.
+                               value:
+                                 code: FORMAT_QOPS_REF
+  formatPensionSchemeTaxReference:
+                     responses:
+                       400:
+                         body:
+                           application/json:
+                             type: types.errorResponse
+                             examples:
+                               formatPensionSchemeTaxReference:
+                                 description: The format of pension scheme tax referencer is not valid.
```

Comment:
While this matches the business spec I think there is a typo in the business spec for reference (i.e. not referencer) could you check please?